api/server.py

```diff
@@ -407,28 +463,34 @@ def user_list_users():
             + page
             + "&per_page="
             + per_page
-            + "&include_totals=true"
+            + "&include_totals=true&sort=last_login:-1"
```

Comment:
this sorts the user list by the last login date—something we were previously doing with frontend javascript ✅